CMakeLists.txt

```diff
@@ -42,6 +43,11 @@ include(CTest)
 if (BUILD_TESTING)
     add_subdirectory(tests)
 endif()
+
+# Create distributions.
+set(CPACK_PACKAGE_CONTACT "NOAA EMC")
+set(CPACK_GENERATOR "STGZ;TGZ;TZ;DEB;RPM")
```

Comment:
Generating `DEB` and `RPM` on non-Linux machines doesn't work

Reply:
How do I specify in CMake that something should only  happen on Linux?